lib/acunetix/concerns/cleanup.rb

```diff
@@ -8,32 +8,39 @@ def cleanup_html(source)
 
       format_table(result)
 
-      result.gsub!(/&quot;/, '"')
-      result.gsub!(/&amp;/, '&')
-      result.gsub!(/&lt;/, '<')
-      result.gsub!(/&gt;/, '>')
-
       result.gsub!(/<h[0-9] >(.*?)<\/h[0-9]>/) { "\n\n*#{$1.strip}*\n\n" }
       result.gsub!(/<b>(.*?)<\/b>/) { "*#{$1.strip}*" }
-      result.gsub!(/<br\/>/, "\n")
+      result.gsub!(/<br\/>|<br \/>/, "\n")
       result.gsub!(/<div(.*?)>|<\/div>/, '')
+      result.gsub!(/<span.*?>(.*?)<\/span>/m){"#{$1.strip}"}
+      result.gsub!(/<span.*?>|<\/span>/, '') #repeating again to deal with nested/empty/incomplete span tags
+
+      result.gsub!(/<a (.*?)href='(.*?)'><i(.*?)><\/i>(.*?)<\/a>/m) { "\"#{$4}\":#{$2}" }
```

Comment:
Similarly, this one should cover the next one (L19) so we don't need multiple regex's to parse the same tag.